src/redux/notes/selectors.js

```diff
@@ -0,0 +1,6 @@
+import { createSelector } from 'reselect';
+
+export const loadingSelector = createSelector(
+  state => state.notes.notesLoading || state.notes.saveNoteLoading || state.notes.deleteNoteLoading,
+  deleteNoteLoading => deleteNoteLoading
+);
```

Comment:
Update this on Redux forms pr

```suggestion
export const loadingSelector = createSelector(
  state => state.notes.notesLoading,
  state => state.notes.saveNoteLoading,
  state => state.notes.deleteNoteLoading,
  (notesLoading, saveNoteLoading, deleteNoteLoading) => notesLoading || saveNoteLoading || deleteNoteLoading
);
```